force-app/main/default/lwc/apexImperativeMethod/apexImperativeMethod.html

```diff
@@ -7,12 +7,12 @@
                     onclick={handleLoad}
                 ></lightning-button>
             </p>
-            <template if:true={contacts}>
+            <template lwc:if={contacts}>
                 <template for:each={contacts} for:item="contact">
                     <p key={contact.Id}>{contact.Name}</p>
                 </template>
             </template>
-            <template if:true={error}>
+            <template lwc:elseif={error}>
```

Comment:
Not a big deal but we probably want to be consistent across recipes on whether we use `lwc:elseif={error}` like here or `lwc:if={error}` like in `apexImperativeMethodWithComplexParams.html` for example.

Reply:
Good catch Philippe. I refactored the template structure slightly to make `elseif` consistent across recipes. Please check this PR #776